website/docs/advanced/builder.md

```diff
@@ -0,0 +1,211 @@
+ - `--http-mev-relay` flag
+
+:::info
+
+The [register validator Beacon API](https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Validator/registerValidator) will also stop working with the removal of the `--http-mev-relay` flag and will no longer know the url to mev-boost or the active relay.
+
+:::
+
+The following flags will be disabled after this flag is removed, and can optionally be removed.
+ - `--max-builder-consecutive-missed-slots`
+ - `--max-builder-epoch-missed-slots`
+ - `--local-block-value-boost`
+
+## 3. Is Builder Configured?
+
+Once the appropriate flags are fully removed this check shouldn't pass and will fall back to local execution.
+
+## 4. Local Execution Client: kept the same
+
+The execution client can safely continue to run as is with no changes. Once the validator client and beacon node have their settings changed to local execution that is enough to set back. 
+
+## 5. Builder: remove relay URL
+
+removing the `--http-mev-relay` flag from the beacon node will disconnect the builder. Once removed you can safely turn off your builder related services such as your mev boost or relays.
+
+</TabItem>
+</Tabs>
+
+## Frequently asked questions
+
+**Q:** What are the risks of running Prysm with a custom builder instead of using local execution?
+
+**A:** The custom builder whether connected through mev boost or as a relay url will need to be updated consistently with the Prysm adding to another layer of complexity. Depending on the relay used some rewards may be missed due to the relay's connectivity or any builder bugs. Transactions may be censored under certain conditions prior to builders being in protocol through PBS. 
+
+**Q:** Do I need to run my execution client while using a custom builder?
+
+**A:** Yes, the execution client will perform standard tasks and also be the fallback mechanism if the builder is not working correctly. 
+
+**Q:** How do I recover if circuit breaker is triggered?
+
+**A:** Once the circuit breaker is triggered local execution will continue to be used until both conditions max consecutive slots missed, and slots missed in epoch are no longer true. Beacon node does not need to be restarted.
+
+**Q:** What happens if the execution client goes down while connected to the builder?
+
+**A:** The earnings from the local payload will be compared to the earnings from the mev payload and will error if local execution is offline or unavailable.
+
+**Q:** My setup is no-longer using the builder, what happened?
+
+**A:** The relay url may be incorrect, the relay could be down whether outdated compared to your ethereum node setup or a bug, the circuit breaker could have been triggered. 
+
+**Q:** What if the earnings from the builder is lower than the local execution?
+
+**A:** The block from local execution will be used, this could also be triggered though `--local-block-value-boost` if the earnings from the builder doesn't past some percentage threshold.
```

Comment:
```suggestion
**A:** The block from local execution will be used, this could also be triggered through `--local-block-value-boost` if the earnings from the builder don't pass some percentage threshold.
```